tools/wpt/run.py

```diff
@@ -465,6 +465,18 @@ def setup_logging(kwargs, default_config=None):
 
     global logger
 
+    class HasLogHandler(mozlog.handlers.BaseHandler):
+        def __init__(self):
+            self.inner = None
+            mozlog.handlers.BaseHandler.__init__(self, self.inner)
+            self.has_log = False
+
+            # XXX: I have no idea what this second arg does but it's required
```

Comment:
I filed [bugzilla #1515327](https://bugzilla.mozilla.org/show_bug.cgi?id=1515327) for this, given even reading the source I'm not clear what's the right thing to do here.